README.md

```diff
@@ -54,53 +54,16 @@ Features include:
 - Sending of credentials on cross-origin requests (`withCredentials` flag - only has an effect on the client).
 - Request cancellation.
 
+## WebSocket
 
-## WHttp
-`WHttp` acts as an HTTP client that can be used to send many HTTP requests. Client-side this has no effect, but on the
-server this gives you the benefit of cached network connections.
+The WebSocket API mirrors the `dart:io.WebSocket` class to keep things simple. If you've used the server-side `WebSocket`
+class before, then you're ready to go. The benefit is that this same API works for client-side usage as well, which is
+a big improvement over the `dart:html.WebSocket` class.
 
-Additionally, `WHttp` has static methods that make simple HTTP requests easy.
```

Comment:
I know I removed a lot here, but I didn't find it particularly helpful, especially not beyond what's available in the generated docs. 

There needs to be a getting started guide and example code, but I'll address that separately (#49)